docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
       context: .
       dockerfile: docker/development/frontend.Dockerfile
     volumes:
-      - ./frontend:/frontend:cached
```

Comment:
Why do we need this change for the language switcher?

Reply:
> Why do we need this change for the language switcher?

Hm, i guess nope. I thought that only local changes... I repair that